packages/odata-service-writer/src/index.ts

```diff
@@ -117,7 +117,7 @@ async function update(basePath: string, service: OdataService, fs?: Editor): Pro
     const paths = await findProjectFiles(basePath, fs);
     const webappPath = await getWebappPath(basePath, fs);
     ensureExists(webappPath, ['manifest.json'], fs);
-    await enhanceData(basePath, service, fs);
+    await enhanceData(basePath, service, fs, true);
```

Comment:
shouldn't be param for update? As I understand in our UI we have two options - with update and without update